src/orientation/tracker.rs

```diff
@@ -92,4 +92,14 @@ where
     pub fn last_orientation(&self) -> Orientation {
         self.last_orientation
     }
+
+    /// Get access to the underlying accelerometer by borrowing it
+    pub fn accelerometer(&mut self) -> &mut A {
```

Comment:
That would add very much, looking at orientation, that is also an mutable access to the accelerometer, and almost every accelerometer needs mutable access to be able to speak to the underlying peripheral (I2C, SPI).

If we question the design, then having the tracker object own the accelerometer is both good and bad. Good because its easy and portable, bad because it locks the accelerometer inside something that is maybe not very interesting in that specific case ( handling a fall event interrupt  in the accelerometer )

Reply:
Yeah, perhaps the tracker shouldn't own the accelerometer, but borrow it mutably every time you want to obtain an orientation reading.

Reply:
Yeah, maybe, it would at least allow for more use cases, like if one uses multiple accelerometers, or maybe has an IMU that has both gyro and accelerometer.

Reply:
Looking at the `madgwick` crate for some precedent, it just takes three readings as inputs:

https://docs.rs/madgwick/0.1.1/madgwick/struct.Marg.html#method.update

...so perhaps the tracker could be completely decoupled from the accelerometer type.